.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
unnecessary file. Please list all such folders in .gitignore file

Reply:
I added .idea to the .gitignore file